src/lib/y2partitioner/dialogs/md.rb

```diff
@@ -142,6 +142,17 @@ def items
           ]
         end
 
+        def help
+          # TRANSLATORS: Help text heading
+          "<p><b>" + _("RAID Type:") + "</b><ul><li>" +
+            [raid0_help,
```

Comment:
NP: `Yast::HTML.Para` and `Yast::HTML.List` could be used instead of write plain HTML (but I am fine as it is).

Reply:
Since we are using literal HTML tags all over the place everywhere else (in particular in the help texts), this would just be an exercise in futility IMHO.